token-service/src/main/java/org/p2p/token/service/api/TokenServiceApiRepository.kt

```diff
@@ -0,0 +1,8 @@
+package org.p2p.token.service.api
+
+import org.p2p.core.rpc.JsonRpc
+import org.p2p.token.service.model.TokenServiceResult
+
+interface TokenServiceApiRepository {
+    suspend fun <P, T> launch(request: JsonRpc<P, T>): TokenServiceResult.Success<T>
```

Comment:
you can't return only success result here, there is not point in TokenServiceResult then